dashboards/ingress-nginx/ingress-nginx-prometheus.json

```diff
@@ -0,0 +1,432 @@
+                "dimensions": [],
+                "measures": [],
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "nginx_ingress_controller_nginx_process_resident_memory_bytes{${Cluster},${Location},${Namespace}}",
+                  "unitOverride": ""
+                }
+              }
+            ],
+            "thresholds": [],
+            "timeshiftDuration": "0s",
+            "yAxis": {
+              "label": "",
+              "scale": "LINEAR"
+            }
+          }
+        }
+      },
+      {
+        "xPos": 24,
+        "yPos": 20,
+        "width": 24,
+        "height": 16,
+        "widget": {
+          "title": "CPU Usage",
+          "xyChart": {
+            "chartOptions": {
+              "mode": "COLOR"
+            },
+            "dataSets": [
+              {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "rate (nginx_ingress_controller_nginx_process_cpu_seconds_total{${Cluster},${Location},${Namespace}}[2m])",
+                  "unitOverride": ""
+                }
+              }
+            ],
+            "thresholds": [],
+            "timeshiftDuration": "0s",
+            "yAxis": {
+              "label": "",
+              "scale": "LINEAR"
+            }
+          }
+        }
+      },
+      {
+        "yPos": 20,
+        "width": 24,
+        "height": 8,
+        "widget": {
+          "title": "Upstream Latency Seconds",
+          "xyChart": {
+            "chartOptions": {
+              "mode": "COLOR"
+            },
+            "dataSets": [
+              {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "prometheusQuery": "nginx_ingress_controller_ingress_upstream_latency_seconds_sum{${Cluster},${Location},${Namespace}}",
+                  "unitOverride": ""
+                }
+              }
+            ],
+            "thresholds": [],
+            "yAxis": {
+              "label": "",
+              "scale": "LINEAR"
+            }
+          }
+        }
+      },
+      {
+        "yPos": 8,
+        "width": 8,
+        "height": 8,
+        "widget": {
+          "title": "Active Connections",
+          "scorecard": {
+            "sparkChartView": {
+              "sparkChartType": "SPARK_LINE"
+            },
+            "thresholds": [],
+            "timeSeriesQuery": {
+              "outputFullDuration": true,
+              "prometheusQuery": "nginx_ingress_controller_nginx_process_connections{state=\"active\",${Cluster},${Location},${Namespace}}",
```

Comment:
could you wrap the scorecard queries in `sum(`? Some of them already have it. If there are multiple data points here (for example, nginx ingress is running in multiple clusters, and no cluster filter is applied), I see this error. But sum fixes it.

![AhihJzrBRQrQgTU (3)](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/assets/509446/446ffd62-27ae-44f2-bdf4-19d7a7cb661c)

Reply:
Hey John! I was out sick for the majority of last week, but I'm getting back into the swing of things. This should be good to go. Hope all is well!

Reply:
Hope you're feeling well, thanks for the fixes!